### PR TITLE
thbook: fix some typos and add Ubuntu 24.10 prerequisites

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: thbook/ch05.tex [hunk 1]
```diff
@@ -564,7 +564,7 @@
 You may fill them in three ways:
 
 \list
- * fill a uniform or randomised grid in a temporary picture
+ * fill a uniform or randomized grid in a temporary picture
```
Comment (speleo3) [inline thbook/ch05.tex:567]: British vs. American English

File: thbook/ch06.tex [hunk 2]
```diff
@@ -459,7 +484,7 @@
 
   |ignore| silently lets Proj to choose other transformation which doesn't use the missing
   grid(s); this usually leads to decreased transformation accuracy
-  (say metres instead of centimetres). See the |log| file for a list of
+  (say meters instead of centimeters). See the |log| file for a list of
```
Comment (speleo3) [inline thbook/ch06.tex:487]: British vs. American English

**Comment by mbudaj** [comment]
Indeed, British English shouldn't be changed to American.

**Comment by tristan0x** [comment]
> Indeed, British English shouldn't be changed to American.

Thank you for reviewing the change @speleo3 
It should be good now.

**Pull request merged**